tests/acceptance/features/webUISharingInternalUsers/shareWithUsers.feature

```diff
@@ -9,17 +9,17 @@ Feature: Sharing files and folders with internal users
       | user1    |
       | user2    |
 
-  @skip @yetToImplement
+  @yetToImplement
   @smokeTest
   Scenario: share a file & folder with another internal user
     Given user "user2" has logged in using the webUI
-    When the user shares folder "simple-folder" with user "User One" using the webUI
-    And the user shares file "testimage.jpg" with user "User One" using the webUI
+    When the user shares folder "simple-folder" with user "user1" using the webUI
```

Comment:
are the users not created with display name or cannot we share using the display name?

Reply:
Currently, we do not create users with a display name.
 In the next PR, we can create a step to implement creating a user with a display name.
And then we can share with the user using the display name.